### ISSUE TITLE
Video capturing not working in latest release 1.12.11

### ISSUE THREAD

**Issue by kevbradwick** [groupon/Selenium-Grid-Extras#314]
Hello,

Following the upgrade to 1.12.11, we notice that video recording no longer works.

```
[pool-2-thread-1] ERROR com.xuggle.xuggler - URL: C:\Users\Administrator\Desktop\selenium-grid\video_output\5dcacedd-889f-42d1-af8f-58075bf78063.temp; Error: could not find output format (../../../../../../../csrc/com/xuggle/xuggler/Container.cpp:513)
[pool-2-thread-2] ERROR com.xuggle.xuggler - URL: C:\Users\Administrator\Desktop\selenium-grid\video_output\2946f188-095c-4cff-8596-5cea4d534973.temp; Error: could not find output format (../../../../../../../csrc/com/xuggle/xuggler/Container.cpp:513)
```

Could it be related to https://github.com/groupon/Selenium-Grid-Extras/commit/98fca6218250d0c87510ac543e7bd1128da6ffa3 ?

**Comment by taren7** [comment]
I am facing the same issue after upgrading to 1.12.11

**Comment by smattt** [comment]
I'm also seeing this issue.

`[pool-2-thread-1] ERROR com.xuggle.xuggler - URL: C:\SeleniumGridExtras\video_output\6ef9866a-44a8-4057-85fe-cc80889e2e8c.temp; Error: could not find output format (../../../../../../../csrc/com/xuggle/xuggler/Container.cpp:513)
[pool-2-thread-2] ERROR com.xuggle.xuggler - URL: C:\SeleniumGridExtras\video_output\753feb7b-002c-492e-b41b-f672c247e29d.temp; Error: could not find output format (../../../../../../../csrc/com/xuggle/xuggler/Container.cpp:513)
[pool-2-thread-2] ERROR com.xuggle.xuggler - URL: C:\SeleniumGridExtras\video_output\6bb6e7c4-1343-4030-be9d-27cc32d27ad5.temp; Error: could not find output format (../../../../../../../csrc/com/xuggle/xuggler/Container.cpp:513)
[pool-2-thread-3] ERROR com.xuggle.xuggler - URL: C:\SeleniumGridExtras\video_output\a6a4bbf5-f4fe-456a-b957-55689d4ea693.temp; Error: could not find output format (../../../../../../../csrc/com/xuggle/xuggler/Container.cpp:513)
[pool-2-thread-3] ERROR com.xuggle.xuggler - URL: C:\SeleniumGridExtras\video_output\09a68792-6a99-4057-94d5-29efd8689d91.temp; Error: could not find output format (../../../../../../../csrc/com/xuggle/xuggler/Container.cpp:513)`

My video recording settings are basic:

`    "video_recording_options": {
      "record_test_videos": "true",
      "videos_to_keep": "100"
    },`

Thanks

**Comment by smccarthy** [comment]
This will be fixed in 1.12.12 . The temp file has to end in .mp4.

**Comment by smccarthy** [comment]
@kevbradwick @taren7 @smattt Please try with https://github.com/groupon/Selenium-Grid-Extras/releases/tag/v1.12.12

**Comment by smattt** [comment]
v1.12.12 resolved all of my issues with video recording, thanks for fixing this!

You've made our SDET incredibly happy :)

**Comment by smccarthy** [comment]
Closing as this is verified fixed.

**Comment by kevbradwick** [comment]
just a quick note to say thanks, the fix in 1.12.12 works perfectly.
